components/Navigation/Navigation.tsx

```diff
@@ -20,29 +21,66 @@ const baseNavItemCss = css({
   fontSize: '$3',
   mt: '$2',
 
-  '&:first-child': {
-    mt: 0,
+  '&::before': {
+    boxSizing: 'border-box',
+    content: '""',
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+    borderRadius: '$3',
   },
-
-  '&:hover': {
-    backgroundColor: '$navButtonHoverBg',
-    color: '$navButtonHoverText',
+  '&::after': {
+    boxSizing: 'border-box',
+    content: '""',
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+    borderRadius: '$3',
   },
+
   '&:focus': {
-    backgroundColor: '$navButtonFocusBg',
     color: '$navButtonFocusText',
-    padding: 'calc($2 - 2px)',
-    border: '2px solid $navButtonFocusBorder',
+    boxShadow: 'inset 0 0 0 2px $colors$navButtonFocusBorder',
+
+    '&::before': {
+      backgroundColor: '$navButtonFocusBg',
+    },
+    '&::after': {
+      backgroundColor: '$navButtonFocusBg2',
+      opacity: 0.05,
+    },
+  },
+
+  '@hover': {
+    '&:hover': {
+      color: '$navButtonHoverText',
+
+      '&::before': {
+        backgroundColor: '$navButtonHoverBg',
+      },
+      '&::after': {
+        backgroundColor: 'navButtonHoverBg2',
+      },
+    },
+  },
+
+  '&:first-child': {
+    mt: 0,
   },
 
   variants: {
     active: {
       true: {
-        backgroundColor: '$navButtonActiveBg',
```

Comment:
I think that's the change related to my bug report

Reply:
Should now be fixed :)